### DISCUSSION TITLE
Some libraries in lib are server-only

### DISCUSSION THREAD

**Discussion by riccardobl** [stackernews/stacker.news#3032] [Ideas]
Follow-up of https://github.com/stackernews/stacker.news/pull/1727#discussion_r1899152047 
I am listing here the libraries in lib/ that cannot be used in the client because they depend on the Node environment.

- `lib/proxy.js` : imports node `https` and `http`
- `lib/cln.js` : imports `lib/proxy`
- `lib/rss.js` imports `api/ssrApollo`
- `lib/lnd.js` needs access to the filesystem

@huumn suggested that this code can go under `api/lib`.

Refactoring now would cause conflicts with the pending PRs, so I am using this issue to track them for a later time when there will be fewer pending changes.

**Comment by huumn** [comment]
I plan on spending most of my January paying off some of our technical debt. Reorging files should definitely be part of that.
